src/components/figures/chart/chartJs/chartjs_waterfall_plugin.ts

```diff
@@ -0,0 +1,70 @@
+import { ChartType, Plugin } from "chart.js";
+
+interface WaterfallPluginOptions {
+  showConnectorLines: boolean;
+}
+
+declare module "chart.js" {
+  interface PluginOptionsByType<TType extends ChartType> {
+    waterfallLinesPlugin?: WaterfallPluginOptions;
+  }
+}
+
+export const waterfallLinesPlugin: Plugin = {
+  id: "waterfallLinesPlugin",
+  beforeDraw(chart, args, options: WaterfallPluginOptions) {
```

Comment:
Maybe add a comment here to explain that this part is only for drawing the connectors ? :)